doc/readthedocs/sphinx/conf.py

```diff
@@ -142,7 +142,7 @@
         f"python3 binary not found, what?  Looked for it at: `python3'; error = {str(e)}"
     ) from None
 
-generated_schema_file = repo_root / 'idd' / 'Energy+.schema.epJSON.in'  # I know this will have CMake placeholders
+generated_schema_file = repo_root / 'Energy+.schema.epJSON'  # I know this will have CMake placeholders
```

Comment:
But up here, it looks like this is going to be trying to find the generated schema in the repository root.  I think this needs a tweak but I'll try it first.